guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,25 @@
+[id="configuring-selinux-to-permit-access-to-repositories-on-custom-ports_{context}"]
+= Configuring SELinux to Permit Access to Repositories on Custom Ports
+
+ifdef::satellite,foreman-el,katello[]
+SELinux permits access of {Project} to repositories only on specific ports.
+By default, connecting to web servers running on the following ports is permitted: 80, 81, 443, 488, 8008, 8009, 8443, and 9000.
+
+endif::[]
+
+
+.Procedure
+. On {Project}, to verify the ports that are permitted by SELinux for repositories, enter a command as follows:
++
+[options="nowrap",subs="+quotes"]
+----
+# semanage port -l | grep ^http_port_t
+http_port_t     tcp      80, 81, 443, 488, 8008, 8009, 8443, 9000
```

Comment:
I think this only applies to Pulp, not the entire project. Foreman only respects `http_cache_port_t` and `squid_port_t` if I read this correctly:
https://github.com/theforeman/foreman-selinux/blob/58f9b9b384d9a12338dfa7700b93c1d53e1e143e/foreman.te#L282-L286

Reply:
but using repos implies you are using Pulp does it not?

Reply:
I can imagine that a user sees "On Satellite" and thinks it applies to Satellite in general. Then goes to Satellite and configures a default proxy and expects it to work.

Also, containers aren't really repositories but there it does apply (since it's Pulp). So I'd avoid `repositories` and stick to `content` or `content sync`.

Reply:
We also have a custom port type named `foreman_port_t` which can be used for exactly this purpose: when you need to add a custom port. But it only works on Satellite Server, not Capsule. So I am good with using `http_port_t`, after all its a HTTP communication.

Reply:
reworded to avoid using term "repositories"